mail_tracking/models/mail_message.py

```diff
@@ -178,10 +188,13 @@ def toggle_tracking_status(self):
         return self.mail_tracking_needs_action
 
     def _get_failed_message_domain(self):
```

Comment:
As you're using it in a `default`, without a record attached, I think this would clarify that:

```suggestion
    @api.model
    def _get_failed_message_domain(self):
```